app/src/main/java/com/spendster/data/entity/User.java

```diff
@@ -5,25 +5,34 @@
 public class User {
     private final int userId;
     private final String email;
-    private final String userName;
+    private final String username;
     private final String password;
     private final double balance;
     private final Date createdDate;
 
     public User(String email, String password){
         this.userId = 0;
         this.email = email;
-        this.userName = null;
+        this.username = null;
```

Comment:
@SimonMaha you should not copy - paste code in constructor. You can call another constructor using "this()"